### ISSUE TITLE
xray_image_classification_processor

### ISSUE THREAD

**Issue by bhaskar2443053** [asyml/ForteHealth#60]
this issue is created to write a processor for x-ray image classification.

we will be using the HuggingFace based VIT pretrained model https://huggingface.co/nickmuchi/vit-finetuned-chest-xray-pneumonia in our task.

we have verified the accuracy claim of 95.51 % for this model on https://data.mendeley.com/datasets/jctsfj2sfn/1 

this processor and reader from issue #59 will help to create an application of x-ray image classification using the ForteHealth pipeline.
